frequency.c

```diff
@@ -0,0 +1,78 @@
+/*Lloyd Page*/
+/*input string, return frequency without printing zeros, repeatability, uses heap sort*/
```

Comment:
Overall, really nice job! The only thing is that it can't handle sentences, but that wasn't necessarily a needed component.